presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeGeneralQueries.java

```diff
@@ -74,11 +74,6 @@ public void testSubqueries() {}
     @Ignore
     public void testTopN() {}
 
-    // This test is flaky: https://github.com/prestodb/presto/issues/19665
```

Comment:
Can you explain why hive.allow-drop-table=true makes testUnionAll to pass? These seem unrelated. Would be nice to provide some context about the change in the commit message and PR description.